### ISSUE TITLE
Should "Aria-hidden with focusable content" require that `aria-hidden` elements not participate in sequential focus navigation order rather than be not focusable?

### ISSUE THREAD

**Issue by kasperisager** [auto-wcag/auto-wcag#374]
I've just discussed this case with @ShadowBB and there is an entirely valid use-case in doing something like the following:

```html
<div class="off-screen" aria-hidden="true">
  <a href="#" tabindex="-1">I'm a link!</a>
</div>
```

The `<a>` element in the example above is still focusable per the HTML specification and nothing stops a developer from programmatically moving focus to the element. However, the intent of `tabindex="-1"` is _usually_ to remove an element from the sequential focus navigation order, i.e. not allow the element to be focused in any way. 

I would therefore think it's a fair assumption to make that when a negative tabindex is used, developers do not intend to move focus to the element if it also has `aria-hidden="true"`. With this assumption in place, the expectation could be changed to only require that elements not be part of the sequential focus navigation order rather than not be focusable.

**Comment by WilcoFiers** [comment]
That's certainly how we implemented it in axe-core.

**Comment by kasperisager** [comment]
It's not how it's currently specified in the rule though, hence this issue.

**Comment by ShadowBB** [comment]
Kasper did convince me that tabindex="-1" doesn't makes an element unfocusable according to the linked definition. So I think the axe-core implementation is incorrect currently (but only because the rule is incorrect and should be changed).

Would referring to the sequential focus navigation order instead of simply "focusable" not fix this rule?

**Comment by kasperisager** [comment]
@ShadowBB The other way around: `tabindex="-1"` _is_ focusable, but is most likely not part of sequential focus navigation order.

**Comment by ShadowBB** [comment]
@kasperisager I agree with this. I think both our statements are in agreement. You state tabindex = "-1" is focusable. I state tabindex= "-1" doesn't make an element unfocusable.

**Comment by kasperisager** [comment]
@ShadowBB You're totally right; I just can't read 🤦‍♂️

**Comment by annethyme** [comment]
@jkodu, I am co-assigning this rule to you since you are the owner of this rule inside the WAI-Tools project. It has a deadline of March 31.

**Comment by annethyme** [comment]
@jkodu, we suggest changing "None of the target elements are focusable, nor do they contain a focusable element." to "None of the target elements participate in [sequential focus navigation](https://www.w3.org/TR/html/editing.html#sec-sequential-focus-navigation), nor do they contain an element that participate in [sequential focus navigation](https://www.w3.org/TR/html/editing.html#sec-sequential-focus-navigation)."

PS:  the word "contain" is also a can of worms.

**Comment by annethyme** [comment]
Being handled by https://github.com/auto-wcag/auto-wcag/pull/439
